### ISSUE TITLE
User should be able to lock the post

### ISSUE THREAD

**Issue by stanriders** [stanriders/circleforms#56]
For example, a tournament registration form should only be available until a certain date. Currently, there's no way to stop people from adding additional answers to the form.

- [ ] When they want to
- [ ] Timed
